src/main/kotlin/no/nav/syfo/domain/Vedtak.kt

```diff
@@ -35,6 +35,8 @@ data class Vedtak private constructor(
         journalpostId = null,
     )
 
+    fun journalfor(journalpostId: JournalpostId): Vedtak = this.copy(journalpostId = journalpostId)
+
```

Comment:
`updatedAt` er kanskje utelukkende noe vi vil lagre i databasen og holde borte fra domenet🤔 Litt usikker selv, bare lurte på det. Det er kanskje naturlig at domenet for eksempel vet når vedtaket sist ble oppdatert? 🤔
Vi kan godt bare ha det sånn det er nå til vi trenger det. Bare tenkte å lufte idéen 😊

Reply:
Ja, hittil har vi vel stort sett brukt det som en database-greie, så tenker vi kan holde det borte fra domenet til vi får behov det ja :+1: